src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -126,7 +126,6 @@ case class AgoraEntity(namespace: Option[String] = None,
 
   AgoraEntity.validate(this) match {
     case Success(_) => this
-    case Failure(errors) => throw new IllegalArgumentException(s"Entity is not valid: Errors: $errors")
```

Comment:
why not remove the call to validate entirely? Seems like just removing the error check makes this a no op anyway